### PR TITLE
Add a Careers page

### PR DESCRIPTION
**Pull request by lachlanhardy**
This PR adds the new Careers page and relevant links to it from around the site (About page, footer, etc).

The work we know about:
- [x] Add Shop link to About page
- [x] Add Careers link to About page
- [x] Add Careers link to Footer
- [x] Add first cut of Careers page
- [x] Add Aspects section to Careers page 
- [x] Add Benefits section to Careers page
- [x] Re-work Bricks component to allow icons and add icons to Benefits section
- [x] Build and style Testimonial section
- [x] Build Values section
- [x] Add to Percy for visual testing
- [x] Build Application Process section
- [x] Build and style Open Positions section
- [x] Check for any components/details I've missed
- [x] Optically adjust each ActionGrid so that images look true
- [x] Check responsive layouts and styles in different browsers
- [x] fix footer layout issue at certain widths
- [x] Add content for Testimonial section
- [x] Add content for Application Process section
- [x] Add content for Benefits section
- [x] Do content revision
- [x] Any other work
- [x] Ensure backend slug validation disallows "/careers"
- [x] Update ALB to route /careers to site (@toolmantim @sj26)

### CODE REVIEW

File: pages/careers/index.js [hunk 1]
```diff
@@ -216,25 +233,25 @@ export default page((props) => (
...
           >
-            <SubSectionHeader>Parental Leave</SubSectionHeader>
+            <SubSectionHeader>Parental leave</SubSectionHeader>
             <CareersParagraph>Vivamus nunc eros, gravida eu tempor non, finibus tempor arcu. Quisque venenatis imperdiet tellus sed varius. Praesent vitae ornare sem.</CareersParagraph>
           </Brick>
           <Brick
             icon={homeOfficeIcon}
           >
-            <SubSectionHeader>Home Office & Coworking</SubSectionHeader>
+            <SubSectionHeader>Home office & co-working</SubSectionHeader>
```
Comment (lachlanhardy) [inline pages/careers/index.js]: Contrary to, well, grammar, the spelling is "coworking". https://en.wikipedia.org/wiki/Coworking

Comment (toolmantim) [reply pages/careers/index.js]: Fair call! Will fix.

**Comment by toolmantim** [comment]
Alright, pushed a fix for yarn audit! This is ready to go 🎉

**Comment by toolmantim** [comment]
2B770553-572F-4C36-9685-C3CB48959ED0

**Comment by lachlanhardy** [comment]
This is the only GIF I have of me and I don't know how to make more. Just assume I'm thinking about this shipping...




lachlan-loves-sharing-pizza

**Comment by toolmantim** [comment]
Actual LOL

**Comment by toolmantim** [comment]
(p.s. Giphy.app for iPhone)

**Comment by sj26** [comment]
That’s a-dough-rable!

**Pull request merged**